### PR TITLE
Simplify Health & Safety

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@o11y.eu>

### CODE REVIEW

File: content/2022-munich/health-and-safety.md [hunk 1]
```diff
@@ -8,51 +8,14 @@ title: Submit Talk
 
 In the interest of keeping our events accessible to as much of the community as possible while still making sure our community is safe and protected, we have updated our Health + Safety policies for PromCon EU 2022.
 
-* All attendees must show proof of COVID-19 vaccination OR (1) negative COVID-19 test upon first entry to the event.
-* We will no longer conduct daily temperature checks upon entry for attendees.
-* <strong>Masks will still be required</strong> at all times unless actively eating or drinking, outdoors, or giving a keynote or breakout talk.
+* <strong>FFP2 Masks will be required</strong> at all times unless actively eating or drinking, outdoors, or giving a keynote or breakout talk.
```
Comment (juliusv) [inline content/2022-munich/health-and-safety.md:11]: Let's say "FFP2 / N95"

File: content/2022-munich/health-and-safety.md [hunk 2]
```diff
@@ -8,51 +8,14 @@ title: Submit Talk
...
-
 ### Mask Mandate
 
 **While many municipalities’ mask mandates have been lifted masks will still be required to be worn at the event.**
 
-Masks must be worn over the nose and mouth and must be made of a tight-knit, non-permeable material. KN95 or N95 masks are recommended. Cloth masks are acceptable. Bandanas, scarves, and gators are not permitted to be used as masks.
+Masks must be worn over the nose and mouth and must be made of a tight-knit, non-permeable material. KN95, N95, or FFP2 masks are required.
 
 Masks do not need to be worn when in outdoor spaces and while actively eating food and beverage indoors when socially distanced from others. Speakers are not required to wear masks while presenting.
-
```
Comment (juliusv) [inline content/2022-munich/health-and-safety.md:26]: I mean, the CNCF (Elna) asked us to add all of this. If we remove it and they are legally the entity behind it, we should at least check with them.

File: content/2022-munich/health-and-safety.md (hunk 1 above)
Comment (roidelapluie) [reply content/2022-munich/health-and-safety.md]: Done!

**Comment by roidelapluie** [comment]
I sent an email to Elna + prom team + promcon organizers

**Review by RichiH — APPROVED** [review]
Given Elna's guidance, this seems OK.

**Pull request merged**